lib/kamal/commands/builder/native/pack.rb

```diff
@@ -0,0 +1,26 @@
+class Kamal::Commands::Builder::Native::Pack < Kamal::Commands::Builder::Native
+  def push
+    combine \
+      pack(:build,
+        config.absolute_image,
+        "--platform", platform,
+        "--builder", pack_builder,
+        buildpacks,
+        "-t", config.absolute_image,
+        "-t", config.latest_image,
+        "--env", "BP_IMAGE_LABELS=service=#{config.service}",
+        secrets.map { |secret| ["--env", Kamal::Utils.sensitive(ENV[secret])] },
```

Comment:
I need to see how this ends up in the final image still, not sure if it's left in the image at the moment.